hub/test/src/testServer.js

```diff
@@ -230,11 +234,11 @@ export function testServer() {
 
     const authorization = `bearer ${authPart}`
     const authenticator = auth.parseAuthHeader(authorization)
-    t.throws(() => authenticator.isAuthenticationValid(testAddrs[1], challengeText),
+    t.throws(() => authenticator.isAuthenticationValid(testAddrs[1], [challengeText]),
```

Comment:
Improving the Flow typing in our tests showed a bug where this test was passing with an unintended error. (`String.includes` vs `Array.includes`)

Reply:
Yep, good catch